cross/owfs/PLIST

```diff
@@ -0,0 +1,24 @@
+bin:bin/owdir
+bin:bin/owexist
+bin:bin/owexternal
+bin:bin/owftpd
+bin:bin/owget
+bin:bin/owhttpd
+bin:bin/owmon
+bin:bin/owpresent
+bin:bin/owread
+bin:bin/owserver
+bin:bin/owtap
+bin:bin/owwrite
+inc:include/owcapi.h
```

Comment:
Header files are needed if you want to install the Python extension to communicate with the server. But I can remove them.